app/bundles/LeadBundle/Translations/en_US/messages.ini

```
@@ -279,7 +278,8 @@ mautic.lead.field.form.properties.lookup="Lookup values"
 mautic.lead.field.form.properties.numberprecision="Precision"
 mautic.lead.field.form.properties.numberrounding="Rounding mode"
 mautic.lead.field.form.properties.select="Options"
-mautic.lead.field.form.isIndex.tooltip="Making this field indexable may improve search performance. %indexCount% indexes out of %maxCount% have been used."
+mautic.lead.field.indexable="Add to Search Index"
```

```suggestion
mautic.lead.field.indexable="Available in Search "
```
@andersonjeccel, Please help me with the label.

---

I think this is better - available in search sounds good.